### DISCUSSION TITLE
Request -- Google Places Auto Complete Example

### DISCUSSION THREAD

**Discussion by b0ot** [quasarframework/quasar#11734] [General - Components / Directives / etc]
Would some kind person be willing to create a simple Google Places autocomplete example with Quasar without any 3rd party library?

I have been wanting a simple auto-complete address capability for a q-input for quite some time but most of the tutorials I have been looking at haven't worked (e.g. [example 1](https://forum.quasar-framework.org/topic/3493/google-autocomplete-places-with-q-input/5) ) or what you to install 3rd party libraries such as  [vue2-google-maps](https://www.npmjs.com/package/vue2-google-maps).

This doesn't seem like something that should require use of 3rd party libraries, especially ones where the main author is begging for help and is stating they won't be able to maintain it moving forward.

**Comment by hawkeye64** [comment]
@b0ot I found this: https://stackoverflow.com/a/63615710/5359558
